CoreFoundation/String.subproj/CFUnicodeDataB.S

```diff
@@ -0,0 +1,20 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+    .global __CFUnicodeDataB
```

Comment:
These asm files should be preprocessed so perhaps we should #ifdef them out to nothing on the platform it is appropriate for e.g. Little endian only has little endian data slices and big endian only has big endian slices. It might make it simpler for detecting duplicate symbols and the such.